api/src/org/apache/cloudstack/api/command/user/network/RestartNetworkCmd.java

```diff
@@ -77,6 +80,13 @@ public Boolean getCleanup() {
         return true;
     }
 
+    public Boolean getMakeRedundant() {
+        if (makeRedundant != null) {
+            return makeRedundant;
+        }
+        return true;
```

Comment:
If we don't pass anything, I think it should be false, it would continue with normal restart instead of this blue-green deployment.

Reply:
Will fix that.

Reply:
Fixed.